test/build/config-out/TEST_BUILD_OUT/src/index.js

```diff
@@ -1 +1 @@
-console.log("fooey");
+console.log('fooey');
```

Comment:
I just run into this myself, the change occurs after running `yarn test`

Reply:
Ha I kept encountering the same 😄